enterprise/coderd/aibridgeproxy_test.go

```diff
@@ -0,0 +1,76 @@
+package coderd_test
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/v2/coderd/coderdtest"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/coder/v2/enterprise/coderd/coderdenttest"
+	"github.com/coder/coder/v2/enterprise/coderd/license"
+	"github.com/coder/coder/v2/testutil"
+)
+
+func TestAIBridgeProxy(t *testing.T) {
+	t.Parallel()
+
+	t.Run("DisabledReturns404", func(t *testing.T) {
```

Comment:
nit: maybe also adding a test for unauthenticated requests would be good to verify the `apiKeyMiddleware` is working